bin/init-lts-line

```diff
@@ -11,22 +11,22 @@ baseline=$1
 
 # Make tagged commit part of the release branch
 branch_name="stable-${baseline}"
-git checkout -b $branch_name jenkins-${baseline}
+# Use reincrementalification commit as branch point: second child of release commit (first is `[maven-release-plugin] prepare for next development iteration`)
+git checkout -b $branch_name $(git log --reverse --ancestry-path --pretty=%H jenkins-${baseline}..master | head -2 | tail -1)
 
 # Update versions
-mvn -q versions:set -DnewVersion=${baseline}.1-SNAPSHOT
+mvn -q versions:set-property -Dproperty=revision -DnewVersion=${baseline}.1
 
-# Point scm tags back to HEAD
-sed -i 's/<tag>.*<\/tag>/<tag>HEAD<\/tag>/' pom.xml */pom.xml
+echo -Plts-release >> .mvn/maven.config
```

Comment:
Automating what `publish-lts-rc.sh` was doing for all branch builds.

(`-Dbuild.type=rc` is useless for Incrementals and support for this was dropped in https://github.com/jenkinsci/jenkins/pull/3430.)